migration/identity-migration/src/test/java/io/camunda/migration/identity/RoleMigrationHandlerTest.java

```diff
@@ -110,37 +99,39 @@ void stopWhenNoMoreRecords() {
     // then
     verify(managementIdentityClient, times(2)).fetchRoles(anyInt());
     verify(roleServices, times(2)).createRole(any());
-    verify(authorizationServices, times(8))
-        .patchAuthorization(patchAuthorizationRequestCaptor.capture());
-    final var authorizations = patchAuthorizationRequestCaptor.getAllValues();
-    final Map<PermissionType, Set<String>> defaultPermissionMap =
-        Arrays.stream(PermissionType.values())
-            .collect(
-                Collectors.toMap(permissionType -> permissionType, permissionType -> Set.of("*")));
-    Arrays.asList(
-            AuthorizationResourceType.PROCESS_DEFINITION,
-            AuthorizationResourceType.DECISION_DEFINITION,
-            AuthorizationResourceType.DECISION_REQUIREMENTS_DEFINITION)
-        .forEach(
-            resourceType ->
-                assertThat(authorizations)
-                    .contains(
-                        new PatchAuthorizationRequest(
-                            1, PermissionAction.ADD, resourceType, defaultPermissionMap)));
-    assertThat(authorizations)
-        .contains(
-            new PatchAuthorizationRequest(
-                1,
-                PermissionAction.ADD,
-                AuthorizationResourceType.APPLICATION,
-                Map.of(PermissionType.ACCESS, Set.of("operate"))));
-    assertThat(authorizations)
-        .contains(
-            new PatchAuthorizationRequest(
-                2,
-                PermissionAction.ADD,
-                AuthorizationResourceType.APPLICATION,
-                Map.of(PermissionType.ACCESS, Set.of("tasklist", "operate"))));
+    // TODO: this part needs to be revisited
```

Comment:
❓ Should we put a GitHub issue link with these `TODO` comments, to try and not forget that we commented this code out?

Reply:
I raised this to @maryarm with the comment above, I don't know in which way she wants to keep track of this 👍